src/ingest_validation_tools/upload.py

```diff
@@ -73,23 +69,66 @@ def __init__(
         self.errors = {}
         self.effective_tsv_paths = {}
         self.extra_parameters = extra_parameters if extra_parameters else {}
-        self.auth_tok = globus_token
-        self.cedar_api_key = cedar_api_key
+        self.globus_token = globus_token
```

Comment:
Just a reminder we need to update ingest-pipeline with the new parameters

Reply:
I think this will address the changed params but let me know if you see anything missing https://github.com/hubmapconsortium/ingest-pipeline/pull/792